tests/server/models/wine-test.js

```diff
@@ -0,0 +1,122 @@
+var dbURI = 'mongodb://localhost:27017/testingDB';
+var clearDB = require('mocha-mongoose')(dbURI);
+
+var sinon = require('sinon');
+var expect = require('chai').expect;
+var mongoose = require('mongoose');
+var Promise = require('bluebird')
+
+// Require in all models.
+require('../../../server/db/models');
+
+var Wine = mongoose.model('Wine');
+var Review = mongoose.model('Review');
+var User = mongoose.model('User');
+
+describe('Wine model', function () {
+
+    beforeEach('Establish DB connection', function (done) {
+        if (mongoose.connection.db) return done();
+        mongoose.connect(dbURI, done);
+    });
+
+    afterEach('Clear test database', function (done) {
+        clearDB(done);
+    });
+
+    it('should exist', function () {
+        expect(Wine).to.be.a('function');
+    });
+
+    describe('Wines', function () {
+
+        
+            var user = new User({
+                email: 'GHA@gmail.com'
+            })
+
+            var wine = new Wine({
+                type: "red",
+                variety: "Malbec",
+                region: "Spain",
+                winery: "L&P",
+                year: 2016,
+                price: 10
+            });
+
+            var review1 = new Review({
+                stars: 3,
+                wine: wine._id,
+                author: user._id
+            });
+
+            var review2 = new Review({
+                stars: 5,
+                wine: wine._id,
+                author: user._id
+            });
+
+        it('should create a new document', function (done) {
+
+            wine.save().then(function (savedWine) {
+                expect(savedWine.type).to.equal('red');
+                expect(savedWine.variety).to.equal('Malbec');
+                expect(savedWine.region).to.equal('Spain');
+                expect(savedWine.winery).to.equal('L&P');
+                expect(savedWine.year).to.equal(2016);
+                expect(savedWine.inventory).to.equal(10);
+                expect(savedWine.size).to.equal(750);
+                expect(savedWine.price).to.equal(10);
+                expect(savedWine.image).to.exist;
+                expect(savedWine.displayName).to.equal("2016 L&P - Malbec");
+                done();
```

Comment:
It tells us in the terminal which test failed since we have an expect. For example it would say expected "savedWine.inventory to equal 10 but got undefined" or something along those lines. Definitely better to split it out, but this was the approach we took to save time.